apps/src/music/views/PatternPanel.tsx

```diff
@@ -17,7 +18,7 @@ import PreviewControls from './PreviewControls';
 import styles from './patternPanel.module.scss';
 
 // Generate an array containing tick numbers from 1..16.
-const arrayOfTicks = Array.from({length: 16}, (_, i) => i + 1);
+const arrayOfTicks = Array.from({length: PATTERN_NUM_EVENTS}, (_, i) => i + 1);
```

Comment:
This just mirrors the existing use of `PATTERN_AI_NUM_EVENTS`, which I found here:

https://github.com/code-dot-org/code-dot-org/blob/mike/pattern-field-validator/apps/src/music/views/PatternAiPanel.tsx#L38

Reply:
Similarly, this should be the "number of events per measure" constant, whatever it gets renamed to.